FSD-5 Shuffling Numbers

AMubinshaikh opened the pull request:
FSD-5 shuffling numbers 
with simple flow
please check.

AMubinshaikh committed:
FSD-5 is completed and ready to merge with  Develop branch

FSD-5  is based on shuffling arrays
with simple flow
please check.

abdulmuheet commented on ArraysShuffling/ShufflingProgram1.cs:
```diff
@@ -0,0 +1,48 @@
+// Write a program in C# language for the below problems.
```
Check the filename, why it's named like this..

abdulmuheet reviewed (changes requested):
Please see the comments provided in the previous PR.

AMubinshaikh committed:
FSD-5 all changes are done

All changes are done whis is provided in PR's link comments kindly check.

AMubinshaikh committed:
Changes are done.

All changes are done according to comments and remove all console inputs and add parameters passing data to finnctions.

AMubinshaikh committed:
Changes are done according to PR Comments

Change variable name and indentation are done.

abdulmuheet commented on ArraysShuffling/ArrayShuffle.cs:
```diff
@@ -0,0 +1,48 @@
+        public void AcceptArray()
+        {
+         Console.WriteLine("Enter array size");
+         arraySize=Convert.ToInt32(Console.ReadLine());
```
Why still reading it from the console? I have already mentioned what is expected, Kindly correct it, even console.writeline is not required.

abdulmuheet commented on ArraysShuffling/ArrayShuffle.cs:
```diff
@@ -0,0 +1,48 @@
+         }
+         
+        }
+        public void DisplayShuffledElements()
```
Displaying stuff should be part of the caller end(program.cs) get it removed from class.

abdulmuheet commented on ArraysShuffling/ArrayShuffleBasic.cs:
```diff
@@ -0,0 +1,44 @@
+        public void AcceptArray()
+        {
+         Console.WriteLine("Enter array size");
+         arraySize=Convert.ToInt32(Console.ReadLine());
```
The same mistake, reading from the console.

abdulmuheet commented on ArraysShuffling/ArrayShuffleBasic.cs:
```diff
@@ -0,0 +1,44 @@
+                arrayShuffle[index] = array[internalLoop];
+                index++;  
+            }
+        for(arrayLoop = 0; arrayLoop<arraySize; arrayLoop++)
```
Identation is messed up.

abdulmuheet commented on ArraysShuffling/ShufflingProgram2.cs:
```diff
@@ -0,0 +1,44 @@
+using System;
+namespace ArraysShuffling
+{
+ public class ShufflingProgram2
```
Check the filename, why it's named like this..

abdulmuheet commented on ArraysShuffling/ShufflingProgram2.cs:
```diff
@@ -0,0 +1,44 @@
+        public void AcceptArray()
+        {
+         Console.WriteLine("Enter array size");
+         arraySize=Convert.ToInt32(Console.ReadLine());
```
Don't read from Console.

abdulmuheet reviewed (changes requested):
Please go through it thoroughly and reply to each comment once done.

AMubinshaikh committed:
Delete ArrayShuffle.cs

AMubinshaikh committed:
Delete ArrayShuffleBasic.cs

AMubinshaikh committed:
Delete ShufflingProgram1.cs

AMubinshaikh committed:
Update and rename ShufflingProgram2.cs to Shuffle.cs

AMubinshaikh committed:
Update Shuffle.cs

AMubinshaikh committed:
Delete Shuffle.cs

AMubinshaikh committed:
Update Program.cs

AMubinshaikh committed:
Update Shuffling.cs

AMubinshaikh committed:
Changes are done

All number are passing through function and that function generate result according to array list and passed to main method
